priority-queue-sf/PriorityQueue.Sender/Fabric.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Fabric;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Services.Runtime;
+using PriorityQueue.Shared;
+using System.Collections.ObjectModel;
+using System.Fabric.Description;
+using Microsoft.ServiceBus.Messaging;
+using System.Diagnostics;
+using System.Linq;
+
+namespace PriorityQueue.Sender
+{
+    /// <summary>
+    /// An instance of this class is created for each service instance by the Service Fabric runtime.
+    /// </summary>
+    internal sealed class Fabric : StatelessService
+    {
+        private QueueManager queueManager;
+
+        public Fabric(StatelessServiceContext context)
+            : base(context)
+        { }
+
+        /// <summary>
+        /// This is the main entry point for your service instance.
+        /// </summary>
+        /// <param name="cancellationToken">Canceled when Service Fabric needs to shut down this service instance.</param>
+        protected override async Task RunAsync(CancellationToken cancellationToken)
+        {
+            await Task.Yield();
+
+            // Get ServiceBus ConnectionString from configuration (Settings.xml)
+            ConfigurationPackage configPackage = this.Context.CodePackageActivationContext.GetConfigurationPackageObject("Config");
+
+            // Access Settings.xml
+            KeyedCollection<string, ConfigurationProperty> parameters = configPackage.Settings.Sections["ConfigSection"].Parameters;
+
+            var serviceBusConnectionString = parameters["ServiceBus.ConnectionString"]?.Value;
+            var topicName = parameters["TopicName"]?.Value;
+
+            this.queueManager = new QueueManager(serviceBusConnectionString, topicName);
+            await this.queueManager.SetupTopicAsync()
+                .ConfigureAwait(false);
+
+            //Test message send loop
+            while (!cancellationToken.IsCancellationRequested)
+            {
+                try
+                {
+                    // Send a low priority batch
+                    var lowMessages = Enumerable.Range(0, 10)
```

Comment:
On second thought, if this code is just sample code, it doesn't need to meet the bar of reusability. I'm okay with leaving this message generation code in here.